src/plansza.cpp

```diff
@@ -0,0 +1,66 @@
+//funkcje dotyczące planszy
+#include <bits/stdc++.h>
+#include <curses.h>
+#include "plansza.h"
+
+void print(int x, int y, std::string str, int kolor=0)
+{
+    ++x;++y;
+    const char *c=str.c_str();
+    attron(COLOR_PAIR(kolor));
+    mvaddstr(y, x, c);
+    attroff(COLOR_PAIR(kolor));
+}
+
+void print(int x, int y, char znak, int kolor=0)
+{
+    ++x;++y;
+    attron(COLOR_PAIR(kolor));
+    mvaddch(y, x, znak);
+    attroff(COLOR_PAIR(kolor));
+}
+
+void print(int x, int y, std::string str)
+{
+    print(x, y, str, 0);
+}
+
+void print(int x, int y, char znak)
+{
+    print(x, y, znak, 0);
+}
+
+void czyscPlansze(char plansza[][10])
+{
+    for(int i=0; i<10; ++i)
+    {
+        for(int j=0; j<10; ++j)plansza[i][j]=' ';
+    }
+}
+
+void rysujPlansze(char plansza[][10])
```

Comment:
myślę, że kolorki planszy są super, ale nadal moglibyśmy zostawić opisy pól dookoła dla odrobinę większej czytelności

Reply:
i czy da się z tej biblioteki też zrobić siatkę między polami, czy taki wygląd planszy uznajemy za ostateczny (pomijając aspekt jej zmian w fazie gry)?

Reply:
> myślę, że kolorki planszy są super, ale nadal moglibyśmy zostawić opisy pól dookoła dla odrobinę większej czytelności

w sensie legendę? no myślę, że tak

Reply:
> i czy da się z tej biblioteki też zrobić siatkę między polami, czy taki wygląd planszy uznajemy za ostateczny (pomijając aspekt jej zmian w fazie gry)?

no my wszystko robimy na znakach, więc wtedy to byśmy mogli po prostu zrobić przerwy jednego znaku między wszystkim, ale nie wiem czy to by dobrze wyglądało
ale jak chodzi o czytelność, to moglibyśmy potencjalnie zrobić kratkę z podobnych kolorów, które będą oznaczać to samo, czyli woda będzie w kratkę ~jasnoniebiesko-ciemnoniebieską np.

Reply:
> > myślę, że kolorki planszy są super, ale nadal moglibyśmy zostawić opisy pól dookoła dla odrobinę większej czytelności
> 
> w sensie legendę? no myślę, że tak

tak tak legendę

Reply:
> > > myślę, że kolorki planszy są super, ale nadal moglibyśmy zostawić opisy pól dookoła dla odrobinę większej czytelności
> > 
> > 
> > w sensie legendę? no myślę, że tak
> 
> tak tak legendę

done